lib/phoenix/test/conn_test.ex

```diff
@@ -464,7 +479,8 @@ defmodule Phoenix.ConnTest do
   """
   @spec recycle(Conn.t) :: Conn.t
   def recycle(conn) do
-    build_conn()
+    build_conn(conn)
```

Comment:
In any case, we should hold until after v1.3 is out, because this may break the tests of people relying on the old, although arguably worse, behaviour.